vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -3,27 +3,27 @@ sidebarDepth: 2
 ---
 # Build and Publish a Project Bundle
 ## Overview
-This tutorial shows you how to take an existing Entando project directory and deploy it to the Entando Component Repository. This involves building a Docker image from your microservice, creating your Entando bundle, checking your Bundle artifacts into git, and deploying the Entando bundle into Kubernetes.
+This tutorial demonstrates how to deploy an existing Entando project directory into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). This involves building a Docker image from your microservice, creating an Entando Bundle, checking your Bundle artifacts into git, and deploying the Bundle into Kubernetes.
 
-The Entando CLI automates many of the tasks involved in deploying an Entando bundle but you can also choose to perform the tasks manually.
+The Entando CLI automates many of the tasks involved in deploying an Entando Bundle, but you can also choose to perform these tasks manually.
 
 ## Prerequisites
 * Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-environment) to verify you have the prerequisites in place for this tutorial (e.g. Java, npm, git).
 ``` sh
 ent check-env develop
 ```
-* You will also need your git credentials, a git repository, and an Entando instance.
+* You will also need your git credentials, a git repository and an Entando instance.
 * You will also need an Entando project directory, either [developed by hand](./publish-simple-bundle.md) or [generated by the Entando Component Generator](../ms/generate-microservices-and-micro-frontends.md).
 
 ## CLI Steps
 The following steps make use of the Entando `ent prj` command and its publication system (pbs) convenience methods. See the [Manual Steps](#manual-steps) section below for a more detailed description of the underlying tasks.
 
-1. Build the project using the `ent prj` command. This saves you from having to build each part of the project individually. If you are using a project directory with just a bundle child directory (e.g. for a bundle exported from an environment or with hand-built components) then you should skip this step since there's nothing to build.
+1. Build the project using the `ent prj` command. This saves you from having to build each part of the project individually. If you are using a project directory with just a bundle child directory (e.g. for a bundle exported from an environment or with hand-built components), then there is nothing to build and you should skip this step.
```

Comment:
take out 'for' in parenthesis>> (e.g. a bundle exported from an environment or with hand-built components) minor thing but it reads a little awkward